x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -78,26 +81,14 @@ export const endpointMiddlewareFactory: ImmutableMiddlewareFactory<EndpointState
 
     const { getState, dispatch } = store;
 
+    await getEndpointPackageInfo(getState(), dispatch, coreStart);
```

Comment:
I assume there is no way to do this only for when the data is needed in the component? could we trigger off of `userChangedUrl` action?

Reply:
I had originally considered limiting this to the overview page but then I realized that all the tabs except for cases and admin uses the OverviewEmpty component and admin need this info anyways. I guess we could do a if !isCasesPage.

I was actually thinking about just doing a useEffect in OverviewEmpty to dispatch an explicit action just for getting the endpoint package info. decided to do this instead though for the consistency with how admin tab is doing it.